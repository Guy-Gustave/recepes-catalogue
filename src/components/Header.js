import React from 'react';

const Header = () => {
  return (
    <div className="fixed-menu">
      <div className="container center">
        <h2>Recepes calalogue</h2>
      </div>      
    </div>
  );
};

export default Header;