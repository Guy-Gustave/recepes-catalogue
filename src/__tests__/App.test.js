import React from 'react';
import renderer from 'react-test-renderer';
import { Link, BrowserRouter as Router, Switch, Route } from 'react-router-dom';
import Header from '../components/Header';

it('renders correctly', () => {
  const tree = renderer
    .create(
      <div className="App">
        <Router>
          <Header />
        </Router>
      </div>,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
