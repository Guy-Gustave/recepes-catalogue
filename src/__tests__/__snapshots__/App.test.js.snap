// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="App"
>
  <div
    className="fixed-menu"
  >
    <div
      className="container center"
    >
      <h2>
        Recepes calalogue
      </h2>
    </div>
  </div>
</div>
`;
